### ISSUE TITLE
missing `py.typed` file

### ISSUE THREAD

**Issue by DetachHead** [microsoft/appcenter-rest-python#14]
```py
from appcenter import AppCenterClient
```
```
> mypy .
main.py:1:1: error: Skipping analyzing "appcenter": module is installed, but missing library stubs or py.typed marker  [import]
mainpy:1:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

see https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker

**Comment by dalemyers** [comment]
Something else is going on here. This library has full type hinting. We use it in other places without issue. How did you install the library? And what version?

**Comment by DetachHead** [comment]
Version 3.0.0. It's definitely the lack of the `py.typed` file causing the issue, I've verified that my PR fixes it.  

To reproduce:

1. Create a new poetry project
2. Run `poetry add appcenter`
3. Create a new `.py` file with `import appcenter` in it
4. Run `mypy .` - error
5. Run `poetry remove appcenter` (this step is required on poetry <1.2 due to https://github.com/python-poetry/poetry/issues/5051)
6. Change the version of `appcenter` in `pyproject.toml` to `appcenter = { git = "https://github.com/DetachHead/appcenter-rest-python.git", rev = "d42cdf7b15d519eebdaa538336f3f9b51a60002e" }
7. Run `poetry update`
8. Run `mypy .` again - no error

`py.typed` is documented in [PEP-561](https://peps.python.org/pep-0561/#packaging-type-information).

Let me know if you need any more info

**Comment by dalemyers** [comment]
You are 100% correct. I looked into this and it seems that where this was used, it was configured to ignore these errors. Thanks for the report and the PR!

**Comment by dalemyers** [comment]
Released as 3.1.0

**Comment by KotlinIsland** [comment]
PEP-561 is so dumb, if you ask me type checkers should at least try to see if it's typed...
